galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.mq.manager/src/main/java/dev/galasa/mq/Queue.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.mq;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import dev.galasa.framework.spi.ValidAnnotatedFields;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.FIELD })
+@MqManagerField
+@ValidAnnotatedFields({ IMessageQueue.class })
+public @interface Queue {
+
+    String queueMgrTag() default "PRIMARY";
+    String name() default "";
+    String tag() default "";
+    String archive() default "true";
```

Comment:
this should not be a String,  should be a Boolean

Reply:
You know - I knew it should be a boolean - but could I get that to compile when I did - nope!  Now of course it's been mentioned it works first time.  Will get that fixed - and thanks for scaring my compiler into submission

Reply:
done